cypress/fixtures/editor/videos/videoListBrightcove.json

```diff
@@ -0,0 +1,687 @@
+[ {
```

Comment:
Er disse ikke recordet?

Reply:
Recorda på nytt i tilfelle api'en har endret seg, for å få nye verdier inn altså holde mest mulig oppdatert